R/utils.R

```diff
@@ -206,6 +206,6 @@ md5sum <- function(path) {
   con <- base::file(path, open = "rb")
   on.exit(close(con), add = TRUE)
 
-  # compute md5 sum of contents
-  as.character(openssl::md5(con))
+  # compute md5 sum of contents and return as ordinary characters
+  unclass(as.character(openssl::md5(con)))
```

Comment:
Interesting. This is to help `jsonlite` deal with the result?

Reply:
Yes. If you give `jsonlite` an object that has an S3 class, it will not serialize it unless it has a serialization method, which is different than `RJSONIO`.

RJSONIO:

    > RJSONIO::toJSON(openssl::md5("Scott"))
    [1] "[ \"bc28af6f750004729474ccbb403bd0ee\" ]"

jsonlite, S3 class `md5`:

    > jsonlite::toJSON(openssl::md5("Scott"))
    Error: No method asJSON S3 class: md5

jsonlite, S3 class `character`:

    > jsonlite::toJSON(unclass(openssl::md5("Scott")))
    ["bc28af6f750004729474ccbb403bd0ee"]